Edge/10.17 rn update (#309)
SoftwareAG/c8y-releasenotes: content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md

---
weight: 60
title: Release 10.17.0
layout: redirect
---

> The Cumulocity IoT Edge release 10.17 includes new features, improvements, fixes and known issues for Cumulocity IoT 10.17. For more information, see [What's new in Cumulocity IoT 10.17](https://www.youtube.com/watch?v=43xcs8iUlOs&list=PLexm-BA6Wrr9nL67HHJLHw3vZBlhTy922) and [Cumulocity IoT 10.17](/release-10-17-0/whatsnew-10-17-0/).

Reviewer: ```suggestion
The Cumulocity IoT Edge release 10.17 includes new features, improvements, fixes and known issues for Cumulocity IoT 10.17. For more information, see the [What´s New] section (/release-10-17-0/whatsnew-10-17-0/) and our video [What's new in Cumulocity IoT 10.17](https://www.youtube.com/watch?v=43xcs8iUlOs&list=PLexm-BA6Wrr9nL67HHJLHw3vZBlhTy922).
```

---
weight: 60
title: Release 10.17.0
layout: redirect
---

> Cumulocity IoT Edge Release 10.17, includes new features, improvements, fixes and known issues Cumulocity IoT Platform 10.17. For more information, see [What's new in Cumulocity IoT 10.17](https://www.youtube.com/watch?v=43xcs8iUlOs&list=PLexm-BA6Wrr9nL67HHJLHw3vZBlhTy922).

Reviewer 2: ```suggestion
The Cumulocity IoT Edge release 10.17 includes new features, improvements, fixes and known issues for the Cumulocity IoT Platform 10.17. For more information, see [What's new in Cumulocity IoT 10.17](https://www.youtube.com/watch?v=43xcs8iUlOs&list=PLexm-BA6Wrr9nL67HHJLHw3vZBlhTy922).
```

> The Cumulocity IoT Edge release 10.17 includes new features, improvements, fixes and known issues for the Cumulocity IoT Platform 10.17. For more information, see [What's new in Cumulocity IoT 10.17](https://www.youtube.com/watch?v=43xcs8iUlOs&list=PLexm-BA6Wrr9nL67HHJLHw3vZBlhTy922).

Reviewer: ```suggestion
The Cumulocity IoT Edge release 10.17 includes new features, improvements, fixes and known issues for Cumulocity IoT 10.17. For more information, see [What's new in Cumulocity IoT 10.17](https://www.youtube.com/watch?v=43xcs8iUlOs&list=PLexm-BA6Wrr9nL67HHJLHw3vZBlhTy922).
```

> Download the Cumulocity IoT Edge VM based distribution version 10.17 from the [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.

Reviewer: ```suggestion
Download the Cumulocity IoT Edge VM-based distribution version 10.17 from the [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.
```

> ## Cumulocity IoT Edge Appliance VM

Reviewer: We do not use heading level 2, please change all headings accordingly.

Author: Done.

### Mongo DB update

MongoDB has been updated to version 5.0.14.

### Known issues

|<div style="width:100px">Ticket</div>|<div style="width:200px">Area</div>|Description
|:---|:---|:---

> |CIE-4367|Kubernetes|**Issue:** Kubernetes certificates expire after one year.<br><br>**Workaround:** Diasble the microservice hosting feature and enable it again. See, [Enabling the microservice hosting feature](https://cumulocity.com/guides/edge/configuration/#configuring-microservices).

Reviewer 2: ```suggestion
|CIE-4367|Kubernetes|**Issue:** Kubernetes certificates expire after one year.<br><br>**Workaround:** To Disable and enable the microservice hosting feature again. For further information see [Enabling the microservice hosting feature](https://cumulocity.com/guides/edge/configuration/#configuring-microservices).
```

> |CIE-4367|Kubernetes|**Issue:** Kubernetes certificates expire after one year.<br><br>**Workaround:** To Disable and enable the microservice hosting feature again. For further information see [Enabling the microservice hosting feature](https://cumulocity.com/guides/edge/configuration/#configuring-microservices).

Reviewer: ```suggestion
|CIE-4367|Kubernetes|**Issue:** Kubernetes certificates expire after one year.<br><br>**Workaround:** Disable and enable the microservice hosting feature again. For further information see [Enabling the microservice hosting feature](https://cumulocity.com/guides/10.17.0/edge/configuration/#configuring-microservices).
```

> |CIE-4376|UI|**Issue:** When switching to a language other than English, translations for the security menu and security page are not available.

Reviewer 2: ```suggestion
|CIE-4376|UI|**Issue:** When switching to a language other than English, the translations for the security menu and security page are not available.
```

> For new enhancements in Cumulocity IoT, see [What's new](/release-10-17-0/whatsnew-10-17-0/).

Reviewer: There is a similar statement above linking to the youtube video. Perhaps link to both places above?

Author: Added this link along with YouTube link.

## Cumulocity IoT Edge on Kubernetes - Public Preview

Cumulocity IoT Edge can now be deployed and managed on a single node Kubernetes cluster using Cumulocity IoT Edge Kubernetes Operator (Edge Operator).

Edge on Kubernetes managed by the Edge Operator enables you with:

- **Unified Management**: Manage Edge through an Edge Custom Resource (CR) in consistent with standard Kubernetes resources. This approach provides a unified and familiar interface for administrators and operators.

> - **Desired State Management**: Specify the desired state of Edge through Edge CR. The Edge Operator then ensures that the actual state matches the desired state, simplifying operations and reducing the need for manual intervention.

Reviewer 2: ```suggestion
- **Desired State Management**: Specify the desired state of Cumulocity IoT Edge through Edge CR. The Edge Operator then ensures that the actual state matches the desired state, simplifying operations and reducing the need for manual intervention.
```

> - **GitOps Friendly**: The declarative nature of CR aligns seamlessly with GitOps workflows. The entire Edge deployment state, including configurations, can be stored and versioned in Git, promoting transparency and traceability. 

Reviewer 2: ```suggestion
- **GitOps Friendly**: The declarative nature of CR aligns seamlessly with GitOps workflows. The entire Cumulocity IoT Edge deployment state, including configurations, can be stored and versioned in Git, promoting transparency and traceability. 
```

> - **Automation**: Edge CR empowers you to automate various operational aspects of Edge like scaling, upgrades, and configuration.

Reviewer 2: ```suggestion
- **Automation**: Edge CR empowers you to automate various operational aspects of Cumulocity IoT Edge like scaling, upgrades, and configuration.
```

> - **Distribution**: Distribute and replicate the Edge deployments using Edge CR. Simplifies the deployment and accelerates the adoption of Cumulocity IoT platform across diverse environments.

Reviewer 2: ```suggestion
- **Distribution**: Distribute and replicate Cumulocity IoT Edge deployments using the Edge CR. This simplifies the deployment and accelerates the adoption of the Cumulocity IoT platform across different environments.
```

> For more information about how to install and manage Cumulocity IoT Edge on Kubernetes, see [Cumulocity IoT Edge Kubernetes Operator](https://cumulocity.com/guides//edge-k8s/k8-edge-introduction/).

Reviewer 2: ```suggestion
For more information on how to install and manage Cumulocity IoT Edge on Kubernetes, see [Cumulocity IoT Edge Kubernetes Operator](https://cumulocity.com/guides//edge-k8s/k8-edge-introduction/).
```

{{< c8y-admon-info >}}

> Public Preview: Functionality is provided with LIMITED support. Customers can open support cases, file bugs and request feature enhancements, which will be taken care of by the Support Team.

Reviewer 2: ```suggestion
**Public Preview**: This functionality is only provided with limited support. Customers can open support cases, file bugs and request feature enhancements, which will then be taken care of by the Support Team.
```

{{< /c8y-admon-info >}}

> Future releases of Edge on Kubernetes include:

Reviewer 2: ```suggestion
Future releases of Cumulocity IoT Edge on Kubernetes include:
```

> - Cumulocity IoT DataHub Integration

Reviewer 2: ```suggestion
- The integration of Cumulocity IoT DataHub
```

> - Messaging Service and the microservice-based data broker

Reviewer 2: ```suggestion
- The messaging service and the microservice-based data broker
```
